irods_consortium_continuous_integration_test_hook.py

```diff
@@ -12,7 +12,7 @@ def Indexing_PackageName_Regex( package_ext, technology = 'elasticsearch' ):
     tech = re.escape(technology)
     ext = re.escape(package_ext)
     return re.compile(
-        r'irods-rule-engine-plugin-(document-type|{tech}|indexing)-[0-9].*\.{ext}$'.format(**locals())
+        r'irods-rule-engine-plugin-(document-type|{tech}|indexing)[-_][0-9].*\.{ext}$'.format(**locals())
```

Comment:
Is this due to packaging differences between 4.3 and 4.2?

Reply:
The way package names are generated changed in the same way for both branches in these commits:

https://github.com/irods/irods_capability_indexing/commit/6da28046893bc251d856edbe821aa98a3a73cd1f (main)
https://github.com/irods/irods_capability_indexing/commit/45d4703fdfeb0545d96c673bf26b82bc9bee8526 (4-2-stable)

Handling this in the test hook in 4-2-stable was introduced in this commit but never made it to main: 

0a00265cf0a8886c6e55f74f2fafd5c6abdcd2e7 (4-2-stable)

The package name for rpms looks like this:
```
irods-rule-engine-plugin-elasticsearch-4.3.0.1-1.x86_64.rpm
```
The package name for debs looks like this:
```
irods-rule-engine-plugin-indexing_4.3.0.1-1~bionic_amd64.deb
```

So I guess that's why we are handling both dashes and underscores.

Reply:
When we are ready to squash, I'm actually going to cherry-pick the referenced commit so that the change history stays in tact.